video/out/hwdec/dmabuf_interop_wl.c

```diff
@@ -48,6 +71,9 @@ bool dmabuf_interop_wl_init(const struct ra_hwdec *hw,
     if (!ra_is_wldmabuf(hw->ra))
         return false;
 
+    if (strstr(hw->driver->name, "vaapi") != NULL)
```

Comment:
Any reason to not just use `strcmp` here? I know this annoying used to be called `vaapi-egl` for a long time, but I'm pretty sure the name won't be changing anytime soon.

Reply:
> I'm pretty sure the name won't be changing anytime soon

famous last words :)

Reply:
fixed